app/controllers/home_controller.rb

```diff
@@ -45,6 +45,7 @@ def site_map
   
   # donate
   def donate
+    @page_subtitle = "Donate or Volunteer"
```

Comment:
Could you make this a translatable string with the key `page_title`?

Reply:
I put this under `layouts.home.donate.page_title`, but let me know if that's too deeply nested or there's some other convention we should be following.